introduction/introduction.slide

```diff
@@ -0,0 +1,77 @@
+Introduction to Go
+
+Grzegorz Żur
+grzegorz.zur@gmail.com
+
+* Why?
+- no new language
+- concurrency
+- simplicity
+- productivity
+- quick compilation
+- dependency management
+- no IDE
+
+* History
+- 2007 started by Robert Griesemer, Rob Pike, Ken Thompson in Google
+- 2009 publicly available
+- 2012 version 1.0 released
+- 2015 version 1.5 released (Go + assembler)
+- 2016 version 1.6 released (HTTP/2)
+- separate project on Github
+- hundreds of authors including Canonical, Red Hat, IBM, Oracle...
+- design documents also available on GitHub
+
+* What?
+- C-like syntax
+- garbage collected
+- simple static typing
+- values and pointers
+- structures and interfaces
+- first class functions, methods and closures
+- implements Communicating Sequential Processes (Erlang, Occam)
+
+* What?
+- natively compiled code
+- statically compiled code utilizing system calls
+- libraries distributed as source code
+- useful standard library
+- tooling included: compiling, formatting, testing, coverage, benchmarking, documentation, profiling, tracing, downloading code
+- BSD licence
+- [[https://play.golang.org/][Playground!]]
+
+* No!
+- type hierarchy or inheritance
```

Comment:
It might be worth noting that there's composition instead which is preferred patten in OOP anyway :)